src/components/widget/carSlides.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+function CarSlides(){
+    const model=''
```

Comment:
Now that I think about it, these might be better as props rather than states. Check the equivalent Listitemcar component in my PR.

Reply:
You might want to read more about state hooks for function components: <https://reactjs.org/docs/hooks-state.html>